ecodev_front/ids.py

```diff
@@ -23,7 +22,31 @@
 HOME_BUTTON = 'home_button'
```

Comment:
```suggestion
HOME_BUTTON_ID = 'home-button-id'
```

De même, normalement toutes les vars dans ce fichier devrait finir pas _ID nan?